### PR TITLE
Gh 1041 swift pm

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: Multisig/Logic/Models/TokenBalance.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import SwiftCryptoTokenFormatter
+import UIKit
```
Comment (MouazAlzahabi) [inline Multisig/Logic/Models/TokenBalance.swift:11]: Is it needed ?

File: Multisig/Logic/Models/Address.swift [hunk 2]
```diff
@@ -7,6 +7,8 @@
 //
 
 import Web3
+import Foundation
```
Comment (MouazAlzahabi) [inline Multisig/Logic/Models/Address.swift:10]: Is it needed ?

**Review by MouazAlzahabi — APPROVED** [review]

**Comment by DmitryBespalov** [comment]
Oh Yeah! 👍 

Screenshot 2021-04-08 at 17 13 29

**Review by DmitryBespalov — APPROVED** [review]

File: Multisig/Logic/Models/Address.swift (hunk 2 above)
Comment (sche) [reply Multisig/Logic/Models/Address.swift:10]: yes

File: Multisig/Logic/Models/TokenBalance.swift (hunk 1 above)
Comment (sche) [reply Multisig/Logic/Models/TokenBalance.swift:11]: yes

**Pull request merged**